torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2560,6 +2561,15 @@ void DedupModuleUses(Module& module) {
   d.dedup();
 }
 
+void FoldQuantizedPrepackingOps(Module& module) {
```

Comment:
I thought coding convention was camelcase for function names.

Reply:
I guess I also have mixed that up on some of my passes.

Reply:
yeah we are following camelcase for methods and helper functions I think. For jit passes I saw we are using capital case, e.g.: https://bddppq.github.io/codebrowser/pytorch/pytorch/torch/csrc/jit/passes/constant_propagation.h.html